api_tests/lib_sdk/actors/actor.ts

```diff
@@ -159,28 +160,73 @@ export class Actor {
 
     public async accept() {
         if (!this.swap) {
-            throw new Error("Cannot accept inexistent swap");
+            throw new Error("Cannot accept non-existant swap");
```

Comment:
🤡 Replacing a valid word by a made up one?

Reply:
Mhmm... wasn't that a recommendation from @tcharding 🤨